development/nautobot_config.py

```diff
@@ -236,10 +236,9 @@
         "infoblox_username": os.getenv("NAUTOBOT_SSOT_INFOBLOX_USERNAME"),
         "infoblox_verify_ssl": is_truthy(os.getenv("NAUTOBOT_SSOT_INFOBLOX_VERIFY_SSL", True)),
         "infoblox_wapi_version": os.getenv("NAUTOBOT_SSOT_INFOBLOX_WAPI_VERSION", "v2.12"),
-        "ipfabric_api_token": os.getenv("IPFABRIC_API_TOKEN"),
-        "ipfabric_host": os.getenv("IPFABRIC_HOST"),
-        "ipfabric_ssl_verify": is_truthy(os.getenv("IPFABRIC_VERIFY", "False")),
-        "ipfabric_timeout": int(os.getenv("IPFABRIC_TIMEOUT", "15")),
+        "ipfabric_api_token": os.getenv("NAUTOBOT_SSOT_IPFABRIC_API_TOKEN"),
```

Comment:
Please, update https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/development/development.env#L97-L99

Reply:
and https://github.com/nautobot/nautobot-plugin-ssot/blob/develop/development/creds.example.env#L48

Reply:
Updated